examples/nfs-pd/testpd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta1
+desiredState:
+  manifest:
+    containers:
+      - name: testpd
+        image: dockerfile/nginx
+        volumeMounts:
+            # name must match the volume name below
+            - name: nfs
+              mountPath: "/var/www/html/mount-test"
```

Comment:
This shouldn't require the subdirectory "mount-test", as the use case of having nginx's default webroot serve a user's content is perfectly valid.